### PR TITLE
data: Engeniusam

### PR DESCRIPTION
**Pull request by Engeniusam**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5585"></a>

### CODE REVIEW

File: data/Engeniusam.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Samuel Macharia",
+  "type": "personal",
+  "bio": "I am a Frontend Software Engineer.",
+  "avatar": "https://twitter.com/engeniusam/photo",
```
Comment (khushalarora) [inline data/Engeniusam.json:5]: Hi @Engeniusam, thank you for contributing  :tada: 
Please delete this line, it is not required anymore :+1: 
```Suggestion
```

File: data/Engeniusam.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/engeniusam/",
+      "icon": "FaLinkedin"
+    },{
+      "name": "Follow me on Github",
+      "url": "https://github.com/Engeniusam",
+      "icon": "FaGithub"
+    },
+    ,{
```
Comment (khushalarora) [inline data/Engeniusam.json:21]: Also remove this extra  **' Comma '** before the **' opening brace  { '**
```Suggestion
    {
```

File: data/Engeniusam.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "name": "Follow me on Hashnode",
+      "url": "https://hashnode.com/@Engeniusam",
+      "icon": "SiHashnode"
+    },
```
Comment (khushalarora) [inline data/Engeniusam.json:25]: Also remove this  ' Comma ' after the ' closing brace } '
```Suggestion
    }
```

**Review by khushalarora — CHANGES_REQUESTED** [review]

**Review by Engeniusam — COMMENTED** [review]
avatar line removed and the extra commas.

**Review by khushalarora — APPROVED** [review]
Looks good, thanks  :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**